confgenerator/testdata/valid/linux/metrics-receiver_apache_custom/input.yaml

```diff
@@ -14,11 +14,11 @@
 
 metrics:
   receivers:
-    apache_metrics:
+    apache:
       type: apache
-      collection_interval: 30s
+      server_status_url: http://localhost/server-status?auto
```

Comment:
Can we add the `collection_interval` back for this custom test case? With the lowest possible value? I was looking at https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/apache#apache-metrics and it doesn't specify what the lowest value is, is it 30s?

Reply:
Sure thing. Will add it back and push a commit once the integration tests finish (Trying to avoid triggering integration tests again. I'll have the unit tests run)

Reply:
Hmm... I just realized the previous integration tests were not triggered properly. So I just made the change to `10s` (the minimum) and re-triggered them.